### ISSUE TITLE
Improve Verificator Performance

### ISSUE THREAD

**Issue by ckaarle** [ckaarle/ma#24]
- [x] add explicit early stopping
- [x] use BatchNorm
- [x] learning rate
- [x] smaller net (MORPH has 13.000 identities)
- [x] more data examples
- [x] optimizer
- [x] network should be configurable
- [x] check accuracy of same vs. different pairs (increase margin if necessary)
- [x] check way of normalize feature vector

- [x] use small(er) custom net
- [x] test VGGFace (Deep Face Recognition)
- [x] train on un-normalized images
- [ ] use hard different pairs
